src/mpi/mpi_caf.c

```diff
@@ -4454,39 +4454,36 @@ PREFIX (event_wait) (caf_token_t token, size_t index,
 		     int until_count, int *stat,
 		     char *errmsg, int errmsg_len)
 {
-  int ierr=0,count=0,i,image=caf_this_image-1;
-  int *var=NULL,flag,old=0;
-  int newval=0;
+  int ierr = 0, count = 0, i, image = caf_this_image - 1;
+  int *var = NULL, flag, old = 0;
+  int newval = 0;
```

Comment:
I general, I think separating style changes from content changes is *imperative*. However, in this case, the style change was so small, and, at the same time, thoroughly discussed, in the PR text that it was impossible to confuse the two. Under most circumstances, this will probably not be the case. For this case, however, I agree that an exception can be made.